packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.spec.ts

```diff
@@ -519,4 +530,161 @@ describe('rum track view metrics', () => {
       expect(getViewUpdate(2).cumulativeLayoutShift).toBe(0.5)
     })
   })
+
+  describe('scroll metrics', () => {
+    describe('on scroll', () => {
+      let scrollMetrics: ScrollMetrics = {}
+      let stopTrackScrollMetrics: () => void
+      let clock: Clock
+      const getMetrics = jasmine.createSpy('getMetrics')
+
+      const newScroll = (scrollParams: { scrollHeight: number; scrollDepth: number; scrollTop: number }) => {
+        getMetrics.and.returnValue(scrollParams)
+
+        window.dispatchEvent(createNewEvent(DOM_EVENT.SCROLL))
+
+        clock.tick(THROTTLE_SCROLL_DURATION)
+      }
+      describe('with flag enabled', () => {
+        beforeEach(() => {
+          clock = mockClock()
+          addExperimentalFeatures([ExperimentalFeature.SCROLLMAP])
+          stopTrackScrollMetrics = trackScrollMetrics(
+            { relative: 1 as RelativeTime, timeStamp: 2 as TimeStamp },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏If you make the viewStart at 0 you will be able to have `maxScrollDepthTime` that are multiple of 1000 instead of having 999.